### DISCUSSION TITLE
Training videos and/or training documentation

### DISCUSSION THREAD

**Discussion by lynnmiller24** [oqtane/oqtane.framework#4737] [Q&A]
Hello.  I am just getting started with Oqtane.  I have been successful installing it, running it, and creating some simple modules.  My question is are there any additional training resources besides the 7 part series on Youtube (https://www.youtube.com/@oqtane6402/videos) from four years ago and also what I see in docs.oqtane.org?  Did anything come from the Oqtane.Learn proposal?  I just want to make sure I have access to all possible resources as I want to learn Oqtane as fluently as possible.  Any other learning recommendations?

**Comment by thabaum** [comment]
[Oqtane Official Blog](https://www.oqtane.org/Blog) is another source to review.

It really depends on what you are trying to do I guess.  There are a few things contributed by members here in discussions show and tell areas as well I believe.  Things like working with different UI packages like MudBlazor and Syncfusion.

I would also like to watch and review the latest event from Oqtane[Trail Blazor ](https://trailblazor.net/) event which at the bottom is a link to the [YouTube channel](https://www.youtube.com/@Devessence) for more videos that should be added soon (button link at bottom of the page currently goes to the video page where it should be available when created).  There was a lot going on in Florida with storms so this held things back but I am sure this is going to be released sometime in the near future.  I was not able to attend myself due to unexpected work so look forward to reviewing it.

We are trying desperately to get more help with [Oqtane.Docs project](https://github.com/oqtane/oqtane.docs) to help collaborate on ways we can enhance videos and training documentation.

**Reply by lynnmiller24** [reply]
The TrailBlazor event from Oct 3 looks very informative.  I will watch for the session links to be posted.

**Comment by thabaum** [comment]
@lynnmiller24 As for the Oqtane.Learn proposal, not sure if it is the one I had proposed in the past you are referring to.  I think at some point we will create something, but for now Oqtane.Docs is it.

There is also a Discord channel which I hear is very active.  @leigh-pointer manages this area currently and I will see what I can do to include this in our documentation and readme as I will put this on my TODO list in the upcoming week.  I have one other project to bang out which I am hoping I will be done by end of the week to create some breathing room to donate more in this area.

I will try to put in a contribution soon as winter hits I am not as active outdoors... which is when I can contribute more time in this area.

**Reply by thabaum** [reply]
Also you can visit the Oqtane Docs project and feel free to put in requests as issues or discussions there as we can use any feedback to help us direct our efforts to better serve the greater community.

**Reply by lynnmiller24** [reply]
@thabaum , yes it was your proposal for Oqtane.Learn I was referring to.  I had seen it when looking through the Q&A section: https://github.com/oqtane/oqtane.framework/discussions/3633

**Comment by thabaum** [comment]
https://discord.gg/BnPny88avK

There is the discord link that should work currently as I just tested.

**Reply by lynnmiller24** [reply]
Thank you for pointing me to these various additional resources.  I really appreciate it.

**Comment by sbwalker** [comment]
@lynnmiller24 I am planning to revise the Oqtane video series this month

**Reply by lynnmiller24** [reply]
@sbwalker Excellent!
